keps/prod-readiness/sig-auth/5284.yaml

```diff
@@ -0,0 +1,6 @@
+# The KEP must have an approver from the
+# "prod-readiness-approvers" group
+# of http://git.k8s.io/enhancements/OWNERS_ALIASES
+kep-number: 5284
+alpha:
+  approver: "enj"
```

Comment:
Must be one of:


  prod-readiness-approvers:
    - deads2k
    - johnbelamaric
    - jpbetz
    - soltysh
    - wojtek-t

I would suggest posting on #prod-readiness and seeing who has cycles for this.

Reply:
@deads2k agreed to review it offline.